docs/content/en/docs/operator-manual/piped/installation/installing-on-fargate.md

```diff
@@ -0,0 +1,149 @@
+---
+title: "Installing on ECS Fargate"
+linkTitle: "Installing on ECS Fargate"
+weight: 4
+description: >
+  This page describes how to install Piped as a task on ECS cluster backed by AWS Fargate.
+---
+
+## Prerequisites
+
+##### Having piped's ID and Key strings
+- Ensure that the `piped` has been registered and you are having its PIPED_ID and PIPED_KEY strings.
+- If you are not having them, this [page](/docs/operator-manual/control-plane/registering-a-piped/) guides you how to register a new one.
+
+##### Preparing SSH key
+- If your Git repositories are private, `piped` requires a private SSH key to access those repositories.
+- Please checkout [this documentation](https://help.github.com/en/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent) for how to generate a new SSH key pair. Then add the public key to your repositories. (If you are using GitHub, you can add it to Deploy Keys at the repository's Settings page.)
+
+## Installation
+
+- Preparing a piped configuration file as the following:
+
+``` yaml
+apiVersion: pipecd.dev/v1beta1
+kind: Piped
+spec:
+  projectID: {PROJECT_ID}
+  pipedID: {PIPED_ID}
+  pipedKeyData: {BASE64_ENCODED_PIPED_KEY}
+  webAddress: {CONTROL_PLANE_WEB_ADDRESS}
+  # Write in a format like "host:443" because the communication is done via gRPC.
+  apiAddress: {CONTROL_PLANE_API_ADDRESS}
+
+  git:
+    sshKeyData: {BASE64_ENCODED_PRIVATE_SSH_KEY}
+
+  repositories:
+    - repoId: {REPO_ID_OR_NAME}
+      remote: git@github.com:{GIT_ORG}/{GIT_REPO}.git
+      branch: {GIT_BRANCH}
+
+  # Optional
+  # Enable this Piped to handle ECS application.
+  cloudProviders:
+    - name: ecs-dev
+      type: ECS
+      config:
+        region: {ECS_RUNNING_REGION}
+
+  # Optional
+  # Uncomment this if you want to enable this Piped to handle Terraform application.
+  #  - name: terraform-dev
+  #    type: TERRAFORM
+
+  # Optional
+  # Uncomment this if you want to enable SecretManagement feature.
+  # https://pipecd.dev/docs/user-guide/secret-management
+  # secretManagement:
+  #   type: KEY_PAIR
+  #   config:
+  #     privateKeyData: {BASE64_ENCODED_PRIVATE_KEY}
+  #     publicKeyData: {BASE64_ENCODED_PUBLIC_KEY}
+```
+
+- Create a new secret in [SecretManager](https://aws.amazon.com/secrets-manager/) to store above configuration data securely
+
+```console
+aws secretsmanager create-secret --name PipedConfig \
+  --description "Configuration of piped running as ECS Fargate task" \
+  --secret-string file://piped-config.yaml
```

Comment:
Is this format required by `aws` command?
Or just our convention to tell users that this is a file path.

Reply:
I meant the format of the last one:`file://file-name` .

Reply:
it's the AWS command's args convention 👀 https://docs.aws.amazon.com/cli/latest/userguide/cli-usage-parameters-file.html

Reply:
Thanks for the link. Didn't know about that. 👍